src/config.py

```diff
@@ -24,6 +24,7 @@ class Network(Enum):
     GNOSIS = "gnosis"
     ARBITRUM_ONE = "arbitrum"
     BASE = "base"
+    AVALANCHE = "avalanche"
```

Comment:
what is the naming convention we use here btw? It doesn't seem to follow the backend or the Dune convention